conf/modules.d/milter_headers.conf

```diff
@@ -17,7 +17,13 @@ milter_headers {
 
   # Refer to https://rspamd.com/doc/modules/milter_headers.html for information on configuration
 
-  use = [];
+  use = ["remove-header"];
+  routines = {
+    ['remove-header'] = {
```

Comment:
@firedranzer Does it remove the `X-Spam` header for you?
We expect contributors to test their patches before proposing changes.
```
# rspamadm configtest
ucl parser error: error while parsing /usr/local/etc/rspamd/modules.d/milter_headers.conf: line: 22, column: 4 - 'key must begin with a letter', character: '['
syntax BAD
```